source/kraken/apply_disruption.cpp

```diff
@@ -190,11 +202,94 @@ struct add_impacts_visitor : public apply_impacts_visitor {
     }
 
     void operator()(nt::StopPoint* stop_point) {
-        LOG4CPLUS_INFO(log, "Disruption on stop point:" << stop_point->uri << " is not handled");
+
+        using namespace boost::posix_time;
+        using namespace boost::gregorian;
+
+        type::ValidityPattern impact_vp{meta.production_date.begin()}; // bitset are all initialised to 0
```

Comment:
Add a comment : `Computing a validity_pattern of impact used to pre-filter concerned vjs later`